configs/default/FOXE-FOXEERF722V2.config

```diff
@@ -67,7 +67,7 @@ timer B03 AF1
 # pin B03: TIM2 CH2 (AF1)
 
 # dma
-dma ADC 3 1
+dma ADC 3 0
 # ADC 3: DMA2 Stream 1 Channel 2
```

Comment:
This is incorrect now, making it very confusing.

Reply:
When flash the new firmware, and change to Dshot, the problem happened, then I have to import the config file manually, the previous config file is no problem.

Reply:
@Johnsnow-foxeer mikeller was not refering this` dma ADC 3 0` line, instead it was the # ADC 3 : DMA2 Stream 1 Channel 2 does not correspond to its actual assignment. You need to also change this according to a real FC's output.

Reply:
@haslinghuis should be: `# ADC 3: DMA2 Stream 0 Channel 2`

Reply:
@DusKing1 Harvey asked me to help, was the out-of-sync commentary all that's wrong with this PR?

Reply:
@DieHertz: No, what is wrong is that this is simply reverting to a version that is not optimised for SPI DMA that will lead to poor performance for this target. The other problem is the fact that 'Harvey' (@Johnsnow-foxeer) seems to be hellbound to kill progress and refuses to properly test the actual fix (#483) or provide an explanation of what the problem is.

Reply:
@mikeller I tried to reproduce what might happened to his target and this is the detail that I can provide: https://github.com/betaflight/unified-targets/issues/493.

However I don't understand their engineer's "problem description " so I'm not sure what they actually did.

Reply:
I see, did not notice the other PR and discussion, will look into it

Reply:
@DusKing1: Added a comment and a video to #493 - as I'd stated before, what you say gives you a lockup is working fine for me when testing.